geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -45,6 +48,12 @@ app.module.directive('appMymaps', app.mymapsDirective);
 app.MymapsDirectiveController = function($scope, $compile, gettext) {
 
   /**
+   * @type {ol.interaction.Select}
+   *  @export
```

Comment:
Do you really need to export `selectInteraction`? (I don't think you refer to `selectInteraction` from a template.)